index.html

```diff
@@ -884,7 +884,30 @@ <h2>The <code>application/credential-claims-set-1.1+json</code> Media Type</h2>
   </table>
   <div class="issue" data-number="42"></div>
 </section>
+</section>
 
+<section id="appendix">
+  <h2>Appendix</h2>
+  <section>
+    <h3>Example VC Data Model Mapping</h3>
+    <p>This is one possible unidirectional mapping between 2.0 VC-JWTs and the VC Data Model; other such mappings are possible.</p>
+    <pre class="example">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    {
+        "iss": {"@id": "https://www.w3.org/2018/credentials#issuer"},
+        "sub": {"@id": "https://www.w3.org/2018/credentials#credentialSubject"},
+        "nbf": {"@id": "https://www.w3.org/2018/credentials#issuanceDate"}
+    }
```

Comment:
We should make it clear that this member is injected after verify in this example, so as not to confuse people

Reply:
yes, I agree, I will propose some changes.

Reply:
I'd prefer that we inject a `https://www.w3.org/ns/credentials/jwt/v2` context that contains the `iss`, `sub`, and `nbf` term definitions instead of doing it inline such that a JSON-LD processor isn't required to always process this sort of object.